Gds/src/fprime_gds/executables/cli.py

```diff
@@ -498,7 +499,8 @@ def get_parser():
             action="store",
             required=False,
             type=str,
-            help="Root directory of build artifacts, used to automatically find app and dictionary. [default: install_dest field in settings.ini]",
+            help="Root directory of build artifacts, used to automatically find app and dictionary."
+                 "[default: install_dest field in settings.ini]",
```

Comment:
I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.